wrangler-core/src/main/java/io/cdap/wrangler/registry/UserDirectiveRegistry.java

```diff
@@ -228,8 +228,10 @@ public void reload(String namespace) throws DirectiveLoadException {
     }
   }
 
+  @Nullable
```

Comment:
The ArtifactManager should never be null in any case, if this is null, this means udd will never work in pipeline